package.json

```diff
@@ -27,21 +28,24 @@
   ],
   "license": "ISC",
   "dependencies": {
-    "debug": "^2.2.0",
+    "debug": "^3.1.0",
     "extend": "^3.0.0",
-    "in-publish": "^2.0.0",
-    "lodash.find": "^3.2.1"
+    "in-publish": "^2.0.0"
   },
   "devDependencies": {
-    "babel": "^5.8.23",
-    "babel-eslint": "^5.0.4",
-    "coveralls": "^2.11.4",
-    "docpress": "^0.6.13",
```

Comment:
We can keep this for now, since it's devDep and only used for generating the site. I'll make the security changes on that end.